.travis.yml

```diff
@@ -1,9 +1,10 @@
 sudo: false
 language: rust
 cache:
-  cargo: true
   directories:
-   - $HOME/deps
+   - $HOME/.cargo
+before-cache:
+  - rm -rf $HOME/.cargo/registry
```

Comment:
Why do you need to do that?

Reply:
it was sorta sneaked it because everything was broken on Travis, see https://gitter.im/gfx-rs/gfx?at=5c4229f70a491251e31a3355